libs/genetic-algorithm/src/selection/roulette_wheel.rs

```diff
@@ -36,16 +36,9 @@ impl SelectionPolicy for RouletteWheelSelection {
 /// Roulette-wheel selection via stochastic acceptance
 impl Selector for RouletteWheelSelector {
     fn select<'a, I: Individual>(&self, population: &'a [I], rng: &mut ChaCha8Rng) -> &'a I {
-        loop {
-            let individual = population
-                .iter()
-                .choose(rng)
-                .expect("got an empty population");
-
-            if rng.gen_bool((individual.fitness() / self.max_fitness) as _) {
-                return individual;
-            }
-        }
+        return population
```

Comment:
Would you mind dropping this `return`? Usually it's more idiomatic not to use `return` unless it's necessary.

Reply:
Done!